microshift_running_apps/microshift-applications.adoc

```diff
@@ -12,4 +12,9 @@ include::modules/microshift-manifests-overview.adoc[leveloffset=+1]
 
 include::modules/microshift-manifests-override-paths.adoc[leveloffset=+1]
 
+//[id="additional-resources_applications-with-microshift_{context}"]
+[role="_additional-resources"]
+== Additional resources
```

Comment:
This assembly-level additional resources section can't have a module included after it. Basically:

- If you mean to have this section apply to the entire assembly, you need to add it after the module included on line 20, not here.
- If you mean to have this section apply only to `modules/microshift-manifests-override-paths.adoc`, then this can stay here, but it just needs a different format to be a module-level section:
```suggestion
[role="_additional-resources"]
.Additional resources
```

Reply:
done. change is made